test/resources/error.html

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
I know the actual page has these blank lines but could they be removed from this and the test updated accordingly? The reason I ask is it isn't important if there are blank lines but the content is important and those blank lines might be removed at some point without us knowing about it.